src/EditorFeatures/Core/Extensibility/Completion/ExportCompletionProviderAttribute.cs

```diff
@@ -8,13 +8,13 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     [MetadataAttribute]
     [AttributeUsage(AttributeTargets.Class)]
-    internal class ExportCompletionProviderAttribute : ExportAttribute
+    internal class ExportCompletionProviderMef1Attribute : ExportAttribute
```

Comment:
I needed a different name for the editor layer attribute because I needed to create the feature layer attribute for the public API.  It's internal, and the idea is that eventually the remaining editor layer provider goes away so we don't need the attribute at all, so I just wanted the name to highlight that its different that the public API one and why.